fe/fe-core/src/main/java/com/starrocks/planner/FileScanNode.java

```diff
@@ -502,7 +524,7 @@ private void processFileGroup(
         for (int i = 0; i < fileStatuses.size(); ) {
             TBrokerFileStatus fileStatus = fileStatuses.get(i);
             TFileFormatType formatType = formatType(context.fileGroup.getFileFormat(), fileStatus.path);
-            List<String> columnsFromPath = BrokerUtil.parseColumnsFromPath(fileStatus.path,
+            List<String> columnsFromPath = HdfsUtil.parseColumnsFromPath(fileStatus.path,
```

Comment:
Because parseColumnsFromPath doesn't need to communicate with broker / hdfs / s3, it only parses the columns from the path. 
The codes in BrokerUtil and HdfsUtil are same.